### PR TITLE
Update to README to include an improved description of the data regis…

### PR DESCRIPTION
**Pull request by agbeltran**
…try #101

### CODE REVIEW

**Comment by agbeltran** [comment]
@richardreeve I've also changed the `About` section in the repo

File: README.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
 User documentation is available [here](docs/index.md).
 
-Information on how to run the local registry is available at [https://fairdatapipeline.github.io/docs/local_registry/](https://fairdatapipeline.github.io/docs/local_registry/).
+Information on how to run the local registry is available at [the local registry section in the project website](https://fairdatapipeline.github.io/docs/local_registry/).
 
-Alternativley you can run the local registry in a local VM using [Vagrant](https://www.vagrantup.com) with the included Vagrantfile.
+Alternatively, you can run the local registry in a local VM using [Vagrant](https://www.vagrantup.com) with the included Vagrantfile.
 
 Installation, development and maintenance guides are available in the [Wiki](https://github.com/FAIRDataPipeline/data-registry/wiki).
```
Comment (richardreeve) [inline README.md:19]:
```suggestion
Installation, development and maintenance guides, especially for the remote registry, are available in the [Wiki](https://github.com/FAIRDataPipeline/data-registry/wiki).
```

**Review by richardreeve — APPROVED** [review]
Looks like an improvement to me, thanks! Maybe just one edit to stress that the wiki is really about the remote registry installation?

**Comment by agbeltran** [comment]
Sure, can change that now... but we need to move all the documentation to the website anyway

**Comment by richardreeve** [comment]
Yes, it's quite uncoordinated at the moment. My feeling is that we can continue to do this after we submit the paper, since it won't be frozen at that point! I've just noticed another edit though in the meanwhile.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
-The SCRC data registry is a Django website and REST API which is used by the data-pipeline to store metadata about code runs and their inputs and outputs.
+The *FAIR data registry* is a component of the [FAIR data pipeline](https://fairdatapipeline.github.io/) that maintains a metadata catalogue for different data types utilised and generated within a typical epidemiological modelling workflow.
+
+### Implementation 
+
+The *FAIR data registry* is implemented as a [Django](https://www.djangoproject.com/) web application and REST API. The [FAIR data pipeline](https://fairdatapipeline.github.io) uses the data registry to store metadata about code runs and their inputs and outputs. 
+
+### Documentation
 
 User documentation is available [here](docs/index.md).
```
Comment (richardreeve) [inline README.md:13]: I'd move this to the end actually - again it's really for the remote registry, so probably with a comment like the one I added...

```suggestion
```

File: README.md (hunk 1 above)
Comment (richardreeve) [inline README.md:19]:
```suggestion
Installation, development and maintenance guides, especially for the remote registry, are available in the [Wiki](https://github.com/FAIRDataPipeline/data-registry/wiki). User documentation for the remote registry is also available [here](docs/index.md).
```

Comment (richardreeve) [reply README.md:19]: Better suggestion below.

**Pull request merged**